lib/__tests__/validate.test.ts

```diff
@@ -0,0 +1,112 @@
+import fs, { Stats } from 'fs-extra';
+import { validateHubl } from '../../api/validateHubl';
+import { walk } from '../fs';
+import { lint } from '../cms/validate';
+import { LintResult, Validation } from '../../types/HublValidation';
+
+jest.mock('fs-extra');
+jest.mock('../../api/validateHubl');
+jest.mock('../fs');
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const mockedFsStat = fs.stat as jest.MockedFunction<any>;
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const mockedFsReadFile = fs.readFile as jest.MockedFunction<any>;
+const mockedWalk = walk as jest.MockedFunction<typeof walk>;
+const mockedValidateHubl = validateHubl as jest.MockedFunction<
+  typeof validateHubl
+>;
+
+const mockFsStats = jest.createMockFromModule<Stats>('fs-extra');
+
+mockFsStats.isDirectory = jest.fn(() => true);
+
+const mockValidation: Validation = {
+  meta: {
+    all_widgets: [],
+    widgets_in_rich_text: [],
+    editable_flex_areas: [],
+    editable_layout_sections: null,
+    email_style_settings: null,
+    sms_flex_area: [],
+    google_font_variations: null,
+    custom_font_variations: [],
+    has_style_tag: false,
+    has_header_tag: false,
+    output_html: '',
+    has_menu_tag: false,
+    has_theme_setting_function: false,
+    template_source: '',
+    attribute_defaults: null,
+    template_errors: [],
+    path_dependencies: [],
+    theme_field_dependencies: [],
+    template_type_ids: null,
+    exact_path_references: [],
+    required_scopes_to_render: [],
+  },
+  renderingErrors: [],
+  html: '',
+};
+
+describe('lint function', () => {
```

Comment:
Can you update this to include a wrapper `describe()` that includes the path to this file? The current pattern is:
```
path/to/file
  function 1
  function 2
```

Reply:
Addressed in https://github.com/HubSpot/hubspot-local-dev-lib/pull/195/commits/e98326c0500fa5716cd9765c2558f402dabeec3d